config.json

```diff
@@ -1582,6 +1582,18 @@
         ],
         "difficulty": 3
       },
+      {
+        "slug": "pop-count",
+        "name": "Pop Count",
+        "uuid": "8a9d6e1e-9188-44c9-8681-ade84d340dd0",
+        "practices": [
+          "bit-manipulation"
+        ],
+        "prerequisites": [
+          "bit-manipulation"
```

Comment:
I wonder what other concepts are necessary here. Definitely not just bit-manipulation because on the Elixir track, you can learn bit-manipulation together with anonymous functions literally as a second exercise, right after basics.

Your solution uses multiple clause functions, guards, and tail call recursion. I assume it could also be solved with normal recursion? If it could be solved with normal recursion only, that would be better as a prerequisite than tail call recursion as it's easier.

Do you know which language already published this exercise and has some community solutions that I could look at?

Reply:
There are some published [on the Elm track](https://exercism.org/tracks/elm/exercises/pop-count/solutions/
). 
Some use, tail recursion, recursion, some don't use recursion at all (Elm compile to JS, so you know you have at most 32 bits).